File: app/src/main/java/com/xjl/eimui_demo/bean/IUser.java
package com.xjl.eimui_demo.bean;

import com.xjl.eimui.messagelist.bean.EUser;

public class IUser implements EUser {

    private String id;

    private String nickname;

    private String avatar;

    public IUser(String id, String nickname, String avatar) {
        this.id = id;
        this.nickname = nickname;
        this.avatar = avatar;
    }

    @Override
    public String getId() {
        return this.id;
    }

    @Override
    public String getNickname() {
        return this.nickname;
    }

    @Override
    public String getAvatarPath() {
        return this.avatar;
    }
}

File: app/src/main/java/com/xjl/eimui/demo/bean/IUser.kt
package com.xjl.eimui.demo.bean

import com.xjl.eimui.messagelist.bean.EUser

class IUser(
    @JvmField val id: String,
    @JvmField var nickname: String?,
    @JvmField var avatar: String?
) : EUser {

    override fun getID(): String = id

    override fun getNickname(): String? = nickname

    override fun getAvatar(): String? = avatar

}